rgw/admin/caps.go

```diff
@@ -0,0 +1,62 @@
+// +build ceph_preview
+
+package admin
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+// AddUserCap will add the capabilities for a user
+// PREVIEW
+func (api *API) AddUserCap(ctx context.Context, uid, userCap string) ([]UserCapSpec, error) {
+	if uid == "" {
+		return nil, errMissingUserID
+	}
+	if userCap == "" {
+		return nil, errMissingUserCap
+	}
+
+	user := User{ID: uid, UserCaps: userCap}
+	body, err := api.call(ctx, http.MethodPut, "/user?caps", valueToURLParams(user))
+	if err != nil {
+		return nil, err
+	}
+
+	var ref []UserCapSpec
+	err = json.Unmarshal(body, &ref)
+	if err != nil {
+		return nil, fmt.Errorf("%s. %s. %w", unmarshalError, string(body), err)
+	}
+
+	// returns updated list of UserCaps for the user
+	return ref, nil
+}
+
+// RemoveUserCap will remove the capabilities from a user
+// PREVIEW
```

Comment:
```suggestion
// RemoveUserCap removes the capabilities from a user and returns the updated list of UserCaps for the user.
// PREVIEW
```

Reply:
Please add new lines as appropriate, im on the phone right now, where I can't easily see the width.

Reply:
Thanks! Background is, that like this it will be included in the documentation here: https://pkg.go.dev/github.com/ceph/go-ceph@v0.11.0/rgw/admin